tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AWS/SQS/AwsSqsCommon.cs

```diff
@@ -24,9 +24,10 @@ public static Scope CreateScope(Tracer tracer, string operation, out AwsSqsTags
         {
             tags = null;
 
-            if (!tracer.Settings.IsIntegrationEnabled(IntegrationId) || !tracer.Settings.IsIntegrationEnabled(AwsConstants.IntegrationId))
+            if (tracer is null || tracer.Settings.IsIntegrationEnabled(IntegrationId) is false || tracer.Settings.IsIntegrationEnabled(AwsConstants.IntegrationId) is false)
             {
                 // integration disabled, don't create a scope, skip this trace
```

Comment:
(low priority)
These comments are very old and we keep copy/pasting them everywhere, but the "skip this trace" part was always wrong. Do you mind removing that phrase everywhere?
```suggestion
                // integration disabled or Tracer.Instance is null, don't create a scope
```